contracts/helpers/HarvesterSwap.sol

```diff
@@ -0,0 +1,204 @@
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
On ething to have in mind, right now we always fallback to the liquid stablecoin. But we could also rebalance between 2 yield bearing assets directly USDM and steakUSDC for instance